zerver/tests/test_message_fetch.py

```diff
@@ -218,6 +218,10 @@ def test_add_term_using_channels_operator_and_public_operand_negated(self) -> No
             "WHERE (recipient_id NOT IN (__[POSTCOMPILE_recipient_id_1]))",
         )
 
+    def test_add_term_using_with_operator(self) -> None:
+        term = NarrowParameter(operator="with", operand=12)
+        self._do_add_term_test(term, "SELECT id \nFROM zerver_message")
```

Comment:
What are you verifying in this test?

Reply:
oh! this is from the first version of PR which I implemented using a separate query for with operator. Since now it just corrects the narrow, it's of no use. I'll update it.